src/resourceIndex.js

```diff
@@ -41,7 +41,11 @@ class ResourceVersions {
     if (!dateDescriptor.match(/^\d{4}-\d{2}-\d{2}$/)) {
       return false;
     }
-    const availableDates = (this.data?.versions || []).map((v) => v.date);
+    const availableDates = (this.data?.versions || [])
+      .map((v) => v.date)
+      // Index is already sorted but re-sorting asserts this; see
+      // <https://github.com/nextstrain/nextstrain.org/pull/719#discussion_r1361494126>
+      .sort((a, b) => a < b ? 1 : -1);
```

Comment:
This is no longer applicable after the changes related to associating version descriptors to version IDs.

(In a previous conversation on this sorting you mentioned performance, especially as it would pertain to sorting an already sorted list, but `.sort().reverse()` would not have had this nice aspect.)